examples/README.md

```diff
@@ -1,20 +1,17 @@
 # Code examples
```

Comment:
Maybe I am asking for too much to be explained in a single PR, but what kinds of symlinks are exactly created by `examples:install`? 

When I ran `npm run examples:install 11.1.0` many modules were installed, but only 4 symlinks created according to the message in green:

```
	Symlink created for handsontable in /11.1.0/docs/angular/basic-example.
	Symlink created for @handsontable/angular in /11.1.0/docs/angular/basic-example.
	Symlink created for handsontable in /11.1.0/docs/angular/demo.
	Symlink created for @handsontable/angular in /11.1.0/docs/angular/demo.
```

<img width="1371" alt="Screenshot 2022-01-31 at 18 35 47" src="https://user-images.githubusercontent.com/566463/151844468-efb6ccc6-9729-4113-84d7-b793c3b20738.png">

Reply:
Added in https://github.com/handsontable/handsontable/pull/9165/commits/5a486fa53cea65673c24677f8e069adb5002063a#diff-49aaa2819e35a856818ecec8c9fa7e1c79ad028d3f44bd749736353cfb51bac9R82-R87